jupyter/datascience/ubi9-python-3.11/Dockerfile.cpu

```diff
@@ -1,3 +1,16 @@
+######################################################
+# mongocli-builder (build stage only, not published) #
+######################################################
+FROM registry.access.redhat.com/ubi9/go-toolset:latest AS mongocli-builder
+
+ARG MONGOCLI_VERSION=2.0.3
```

Comment:
Not sure where best to store the mongocli version. Maybe the ARG shouldn't have a value set here in the actual Dockerfiles, and be set as part of the build commands? I presume that can be set in Makefiles and/or tekton pipelinerun files as appropriate, but not 100% sure where, so LMK what you think!

Reply:
/lgtm

I don't really have an answer to this. `.tekton/` pipelines are undergoing overhaul currently, to be maintained by devtestops afterwards, so any change going forward would require raising jira ticket to them; that's impractical; cc @MohammadiIram , https://redhat-internal.slack.com/archives/C05TTTYG599/p1747115889072959?thread_ts=1747115307.125289&cid=C05TTTYG599

Reply:
I'm going to resolve this since it's an existing pattern in these Dockerfiles to have "version" ARGs specified with a value directly in the Dockerfile, so it's not something that is specific to this PR. See for example `CODESERVER_VERSION`, `ROCM_VERSION`, `CUDA_VERSION`, etc.